src/models/User.js

```diff
@@ -55,10 +55,10 @@ module.exports = (sequelize) => {
       idGoogle:{
         type: DataTypes.STRING
       },
-      isAdmin: {
-        type: DataTypes.BOOLEAN,
+      rol: {
```

Comment:
Lo de is Admin debes dejarlo, así es taba bien. Además si lo retiras nos dañas las otras rutas que y están listas.